src/docs/backup.dox

```diff
@@ -59,6 +59,10 @@ During the period the backup cursor is open, database checkpoints can
 be created, but no checkpoints can be deleted.  This may result in
 significant file growth.
 
+Additionally, if a crash occurs during the period the backup cursor is open,
+then the system will be restored to the most recent checkpoint prior to the
+opening of the backup cursor, even if later database checkpoints were created.
+
```

Comment:
I think we need the further qualification here "if a crash occurs during the period the backup cursor is open **and logging is disabled**, then ..."